src/monitoring/aws/sqs/monitoring.ts

```diff
@@ -0,0 +1,111 @@
+import { GraphWidget, HorizontalAnnotation, IWidget, Metric } from '@aws-cdk/aws-cloudwatch';
+import { IQueue } from '@aws-cdk/aws-sqs';
+import { Stack } from '@aws-cdk/core';
+import { Monitoring, MonitoringProps } from '../../../core/monitoring';
+import { CountAxis, SizeBytesAxis, TimeSecondsAxis } from '../../../widget/axis';
+import { CommonWidgetDimensions } from '../../../widget/constant';
+import { SectionWidget, SectionWidgetProps } from '../../../widget/section';
+import { SqsMetricFactory } from './metrics';
+
+/**
+ * Properties to create SqsMonitoring.
+ */
+export interface SqsMonitoringProps extends MonitoringProps {
+  queue: IQueue;
+}
+
+/**
+ * Monitoring of SQS Queue.
+ */
+export class SqsMonitoring extends Monitoring {
+  protected readonly section: SectionWidgetProps;
```

Comment:
Do you think `section` belongs to the abstract `Monitoring` class? I am thinking it will be common across.

Reply:
I will definitely think about it, and some more patterns might emerge over time, once more and more monitorings are created. For example, alarm support, etc. Thanks for pointing it out.